src/ui_widgets/PathHandle.gd

```diff
@@ -20,9 +20,12 @@ func set_pos(new_pos: PackedFloat64Array) -> void:
 	if precise_pos != new_pos:
 		var path_attribute: AttributePathdata = element.get_attribute(pathdata_name)
 		var cmd := path_attribute.get_command(command_index)
+		if Input.is_physical_key_pressed(KEY_CTRL):
```

Comment:
Use is_key_pressed(), we're not using physical keys in GodSVG by convention (it's more a thing for games)